resources/html/submit-buttons.html

```diff
@@ -9,6 +9,6 @@
     <input id="clear-form" type="button" value="Clear the form"
            onclick="clear_form();"/>
   </div>
-  <form id="form-data-form" method="post" action="http://127.0.0.1:9334/send">
+  <form id="form-data-form" method="post" action="/send">
```

Comment:
Since I'd like to support multiple methods of using the forms, I think the right approach here is to make the submission mechanism into a function, the behavior of which is controlled by data provided to the form by the integration mechanism.

Ah, I see you did this in a later commit. I think that approach looks good.